osu.Game/Screens/Multiplayer/RoomInspector.cs

```diff
@@ -329,17 +329,16 @@ private void displayBeatmap(BeatmapInfo value)
             if (value != null)
             {
                 coverContainer.FadeIn(transition_duration);
-                coverContainer.Children = new[]
+
+                LoadComponentAsync(new BeatmapSetCover(value.BeatmapSet)
                 {
-                    new AsyncLoadWrapper(new BeatmapSetCover(value.BeatmapSet)
-                    {
-                        RelativeSizeAxes = Axes.Both,
-                        Anchor = Anchor.Centre,
-                        Origin = Anchor.Centre,
-                        FillMode = FillMode.Fill,
-                        OnLoadComplete = d => d.FadeInFromZero(400, Easing.Out),
-                    }) { RelativeSizeAxes = Axes.Both },
-                };
+                    RelativeSizeAxes = Axes.Both,
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    FillMode = FillMode.Fill,
+                    OnLoadComplete = d => d.FadeInFromZero(400, Easing.Out),
+                },
+                bsc => coverContainer.Add(bsc));

```

Comment:
same here

Reply:
also not relevant after these changes, but rather than using acronyms for variables like this we usually resort to things like `cover` or just `b` or `c`.